simapp/test_helpers.go

```diff
@@ -257,6 +259,7 @@ func SignCheckDeliver(
 	chainID string, accNums, accSeqs []uint64, expSimPass, expPass bool, priv ...cryptotypes.PrivKey,
 ) (sdk.GasInfo, *sdk.Result, error) {
 	tx, err := simtestutil.GenSignedMockTx(
+		rand.New(rand.NewSource(time.Now().UnixNano())),
```

Comment:
pass `rand.New(rand.NewSource(time.Now().UnixNano()))`  directly to the function